### PR TITLE
feat(remix-node): add `getFilePath`& `remove` methods to `NodeOnDiskFile`

### PR DESCRIPTION
**Pull request by cstrnt**
Closes: #3021

- [ ] Docs
- [x] Tests

Testing Strategy:

- Added a `test` script in the `remix-node` package (remix-node tests never ran before)
- Added a simple unit test for the `createFileUploadHandler` function

### CODE REVIEW

**Comment by cstrnt** [comment]
@kentcdodds @ryanflorence @mjackson any chance you could take a look at this at some time?

**Comment by machour** [comment]
@cstrnt can you fix conflicts? That would help for the review

**Comment by cstrnt** [comment]
@machour sure thing, PR sat around for some time lol

File: packages/remix-node/__tests__/parseMultipartFormData-test.ts [hunk 1]
```diff
@@ -1,35 +1,74 @@
...
 import { Blob, File } from "@web-std/file";
 
 import { Request as NodeRequest } from "../fetch";
 import { FormData as NodeFormData } from "../formData";
 import { internalParseFormData } from "../parseMultipartFormData";
 import { createMemoryUploadHandler } from "../upload/memoryUploadHandler";
+import {
+  createFileUploadHandler,
+  type NodeOnDiskFile,
+} from "../upload/fileUploadHandler";
```
Comment (MichaelDeBoey) [inline packages/remix-node/__tests__/parseMultipartFormData-test.ts:11]:
```suggestion
import type { NodeOnDiskFile } from "../upload/fileUploadHandler";
import { createFileUploadHandler } from "../upload/fileUploadHandler";
```

File: packages/remix-node/package.json [hunk 2]
```diff
@@ -8,6 +8,9 @@
     "url": "https://github.com/remix-run/remix",
     "directory": "packages/remix-node"
   },
+  "scripts": {
+    "test": "jest"
+  },
```
Comment (MichaelDeBoey) [inline packages/remix-node/package.json:13]: We have this already in the root of the project

```suggestion
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by MichaelDeBoey** [comment]
@cstrnt It seems like something went wrong when rebasing.

**Review by MichaelDeBoey — APPROVED** [review]

**Comment by cstrnt** [comment]
The one failing test doesn't correlate to my code at all. Is it known to be a flaky one?

**Comment by kentcdodds** [comment]
Can you help me understand what the use case for this change is?

**Comment by machour** [comment]
@kentcdodds from the related issue #3021 

> I'm currently fiddling around with Remix and while using unstable_createFileUploadHandler I've noticed that the filePath property on a NodeOnDiskFile is private. I don't think that this is really useful because I want to do some cleanup after using the file.
>
> If it's better to keep the filePath property private, I think it would be nice to add a function called remove or cleanup that unlinks the file in the filesystem.

**Comment by kentcdodds** [comment]
Got it. I'll put this on @mjackson's plate 👍

**Comment by cstrnt** [comment]
@kentcdodds @mjackson small reminder 🤔

**Comment by kentcdodds** [comment]
Sorry, we've been very busy with other priorities. We'll get to this eventually. Thank you for your patience.

**Comment by cstrnt** [comment]
I’m deeply sorry, but I don’t feel like supporting a project led by such toxic lead maintainers

**Pull request closed without merging**